posts/201805.md

```diff
@@ -44,7 +44,7 @@ Obelisk team [posted](https://medium.com/obelisk-blog/obelisk-update-january-may
 
 @SiaBillionaire [estimated](https://decred.slack.com/archives/C3P6R055Z/p1526221654000039) that 7940 DCR1 units were sold in Batches 1-5, while Lynmar13 [shared](https://medium.com/@brian7249694/the-current-state-of-the-obelisk-dcr1-fff97f6877be) his projections of DCR1 profitability ([reddit](https://www.reddit.com/r/decred/comments/8lgbv2/the_current_state_of_the_obelisk_dcr1/)).
 
-New Chinese miner was noticed by our Telegram group. Woodpecker WB2 specs 1.5 TH/s at 1200 W, costs 15,000 CNY (~2,340 USD) and the initial 150 units are expected to ship on Aug 15. ([pow8.com](https://pow8.com/product.php) – [translated](https://translate.google.com/translate?sl=auto&tl=en&hl=en&ie=UTF-8&u=https://pow8.com/product.php))
+A new Chinese miner for sale was noticed by our Telegram group. Woodpecker WB2 specs 1.5 TH/s at 1200 W, costs 15,000 CNY (~2,340 USD) and the initial 150 units are expected to ship on Aug 15. ([pow8.com](https://pow8.com/product.php) – [translated](https://translate.google.com/translate?sl=auto&tl=en&hl=en&ie=UTF-8&u=https://pow8.com/product.php))
```

Comment:
Shall we use "pre-sale" to emphasize it will ship much later?

Reply:
Sounds good